docs/claiming.rst

```diff
@@ -78,23 +82,53 @@ claimed, the more it pays.
 This is a linear transition from getting paid 0% of the total payment if the
 request is claimed at the earliest possible time up to 100% of the total payment
 at the very end of the claim window.  This multiplier is referred to as the
-*payment modifier*.
+*payment modifier*.  Refer to the code block pasted below to see how the smart 
+contract calculates the multiplier. This examples is taken from lines 71 - 79 
+of `RequestScheduleLib.sol`.
+
+
+.. code-block:: solidity
+
+    function computePaymentModifier(ExecutionWindow storage self) 
+        internal view returns (uint8)
+    {        
+        uint paymentModifier = (getNow(self).sub(firstClaimBlock(self)))
+                                .mul(100).div(self.claimWindowSize); 
+        assert(paymentModifier <= 100); 
+
+        return uint8(paymentModifier);
+    }
+
 
 It is important to note that the *payment modifier* does not apply to gas
 reimbursements which are always paid in full.  No matter when a call is
 claimed, or how it is executed, it will **always** provide a full gas
-reimbursement.  The only case where this may end up not being true is in cases
-where the gas price has changed drastically since the time the request was
-scheduled and the contract's endowment is now sufficiently low that it is not
-longer funded with sufficient ether to cover these costs.
-
-For example, if the request has a ``payment`` of 2000 wei, a
+reimbursement.  In the past, this was not always the case since the EAC used 
```

Comment:
Moving this *in the past ...* part into an `.. note::` may produce a more desirable layout/formatting for the docs.

Reply:
Good idea